src/e-patches.js

```diff
@@ -21,9 +26,13 @@ program
 
       // build the list of targets
       const targets = {};
-      const patchesConfig = path.resolve(config.root, 'src', 'electron', 'patches', 'config.json');
-      for (const [key, val] of Object.entries(JSON.parse(fs.readFileSync(patchesConfig)))) {
-        targets[path.basename(key)] = val;
+      const patchesConfig = options.config;
```

Comment:
If the config is a custom config passed by the user, we should validate that the file exists at the specified path and throw if it doesn't!

Reply:
SGTM. Added in 0dd17e674ae82cd0ff1d8670f8ae20a83df35ad5